src/data/modifiers.txt

```diff
@@ -2569,6 +2569,14 @@ Item	replica august scepter	Muscle: +15, Mysticality: +15, Moxie: +15, HP Regen
 Item	replica Operation Patriot Shield	Experience: +3, Maximum HP: +20, Maximum MP: +20, Variable
 Item	reusable shopping bag	Food Drop: +20, Booze Drop: +20, Candy Drop: +20
 Item	rice bowl	Food Drop: +20
+# Roman Candelabra: Maximum HP +<span style="font-family: times new roman">XX</span>
```

Comment:
I was going to flag this and question the syntax but I see it is commented out and so a placeholder.  Carry on.